orderbook/src/api.rs

```diff
@@ -100,35 +99,14 @@ struct ApiMetrics {
     requests_duration_seconds: prometheus::Histogram,
 }
 
-#[derive(Serialize)]
-#[serde(rename_all = "camelCase")]
-struct Error<'a> {
-    error_type: &'a str,
-    description: &'a str,
-}
```

Comment:
Seeing as this type is very specific to the orderbook API, it feels weird to have it in `shared`.